packages/e2e/protractor.conf.ts

```diff
@@ -5,7 +5,7 @@ const chromeOptions = {
   args: ['--window-size=800x600']
 };
 
-if (process.env.GITHUB_WORKFLOW) {
```

Comment:
Why did you change this file? By mistake?

Reply:
I believe you made these changes in 1efbd5bc6eddb139da133613eabbe0df96e12b20.

Reply:
ah wow! I didn't realize it wasn't merged back into master yet.